### ISSUE TITLE
"Something Went Wrong" Error Message in OAuth Flow

### ISSUE THREAD

**Issue by camille-le** [Snapchat/express-4.x-passport-snapchat-example#25]
Hi there, I ran into this error a few times when trying to implement this. I think it would be helpful to add a few troubleshooting steps to the README so other folks can debug the same thing. 

![OAuth Flow](https://github.com/Snapchat/express-4.x-passport-snapchat-example/assets/31285549/7c75c83d-0d31-4e7c-badb-b441f9c1f79a)

1) To test with my account, I needed to add my username as a `Demo User`. This was done by accessing the Developer Portal, then clicking `Global > Setup` for my application and scrolling down to "Demo Users" 

<img width="1059" alt="DemoUser" src="https://github.com/Snapchat/express-4.x-passport-snapchat-example/assets/31285549/f9ced747-93e1-4685-940f-c7016ff36bfc">

2) After, I selected Initial Version for my application. There I needed to input the URI that is used in this repo's sample code. I also needed to make sure both `Display Name` and `Bitmoji Avatar` were selected in order to access this data successfully: 

<img width="1034" alt="URI" src="https://github.com/Snapchat/express-4.x-passport-snapchat-example/assets/31285549/7ce8e511-2c97-4c50-a14c-c42a0563f124">

In short, I am passing in the `callbackURL: 'http://localhost:3000/login/snapchat/callback'` as a URI for OAuth. 
https://github.com/Snapchat/express-4.x-passport-snapchat-example/blob/14126e97c46f984e4d2aec5834eca9de36d8dfeb/server.js#L23C1-L24C1

From completing these steps, I was able to complete this tutorial.

**Comment by EfrenAlmanzalamas** [comment]
Here are a few things to verify in the app created in the Snapchat account: it must be approved by submitting the required information, or if it is not, you must use demo users who should be guest members of your business account, have accepted the invitation, and have access to the app created in the Snapchat developer kit.
